presto-hive/src/test/java/com/facebook/presto/hive/metastore/glue/TestGlueInputConverter.java

```diff
@@ -25,21 +25,22 @@
 import com.facebook.presto.hive.metastore.Table;
 import com.facebook.presto.hive.metastore.glue.converter.GlueInputConverter;
 import com.google.common.collect.ImmutableList;
-import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 import java.util.List;
 
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertNull;
 
+@Test(singleThreaded = true)
 public class TestGlueInputConverter
 {
     private Database testDb;
     private Table testTbl;
     private Partition testPartition;
 
-    @BeforeClass
+    @BeforeMethod
     public void setup()
     {
         testDb = TestMetastoreObjects.getPrestoTestDatabase();
```

Comment:
Why do these need to go in a `setup()` method? Make the fields final and initialize there, instead.

Reply:
The objects are mutable and are modified by the tests